lib/methods/export.js

```diff
@@ -7,7 +7,7 @@ const { uniqBy, uniqueId } = require('lodash');
 
 const MAX_RECURSION = 10;
 
-module.exports = self => {
+module.exports = (self) => {
```

Comment:
Any special reason? (I don't object)

Reply:
None, for consistency, I thought we used to do this way. But not in this project, reverting.